### PR TITLE
#157 - Enable full-text search on internal KBs

### PR DESCRIPTION
**Pull request by reckart**
- Improved/updated documentation

### CODE REVIEW

**Comment by Rentier** [comment]
@reckart Issues found

File: inception-ui-kb/src/main/resources/META-INF/asciidoc/user-guide/projects_knowledge-base.adoc [hunk 1]
```diff
@@ -17,7 +17,19 @@ For remote KBs, INCEpTION provides the user with some pre-configured knowledge b
 [.thumb]
 image::kb2.png[align="center"]
 
-In the next step, the user is offered a configurable mapping which relies on a minimal set of IRIs that must be defined for any KB used with the platform. The user can choose one of the pre-configured mapping or provide her own custom defined mapping as shown in the figure below. In the advanced settings, the user can leverage this feature of KB settings when one doesn't want the entire knowledge base to be used and rather choose to identify some specific root concepts. This feature specially helps in case of large knowledge bases such as Wikidata.
+=== Schema mapping
+
+Different types of knowledge base schemata are supported via a configurable mapping mechanism. The user can choose one of the pre-configured mapping or provide her own custom defined mapping as shown in the screenshot below. 
+
+In the advanced settings, the user can leverage this feature of KB settings when one doesn't want the entire knowledge base to be used and rather choose to identify some specific root concepts. This feature specially helps in case of large knowledge bases such as Wikidata.
```
Comment (Rentier) [inline inception-ui-kb/src/main/resources/META-INF/asciidoc/user-guide/projects_knowledge-base.adoc]: I do not understand 
doesn't want the entire knowledge base to be used

Is there something missing about the concept tree?

File: inception-ui-kb/src/main/resources/META-INF/asciidoc/user-guide/projects_knowledge-base.adoc [hunk 2]
```diff
@@ -17,7 +17,19 @@ For remote KBs, INCEpTION provides the user with some pre-configured knowledge b
...
  
 [.thumb]
-image::kb3.png[align="center"]
\ No newline at end of file
+image::kb3.png[align="center"]
+
+=== Full text search
+
+Full text search support in the knowledge base enables a more efficient access to the knowledge
```
Comment (Rentier) [inline inception-ui-kb/src/main/resources/META-INF/asciidoc/user-guide/projects_knowledge-base.adoc]: Full text search in knowledge bases enables searching for entities by for instance their label. This is a prequesite ...

**Comment by reckart** [comment]
@Rentier there are many parts of the documentation which need improvement and revision. I currently do that opportunistically and incrementally whenever I come across a section that could use improvement and if I have a good idea how to improve it.

In order to wrap up the 0.6.0 release, my suggestion would be that you incorporate your improvement suggestions for the time being. 

Then we should maybe to a half-day session some time in the next weeks where we read through the documentation and improving it in a more systematic way?

**Pull request merged**